### PR TITLE
Update to gSOAP 2.8.98

### PR DESCRIPTION
**Pull request by madduci**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ConanFileDefault(ConanFileBase):
 
     def requirements(self):
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1d")
+            self.requires("OpenSSL/1.1.1c@conan/stable")
```
Comment (uilianries) [inline conanfile.py:22]: Why?

Comment (madduci) [reply conanfile.py:22]: it looks like `openssl/1.1.1d@conan/stable` isn't available anymore on conan center, that's why I stepped back to 1.1.1c

Comment (uilianries) [reply conanfile.py:22]: Use from Conan Center Index. It's the official now and OpenSSL form Conan Community is totally deprecated.

Comment (madduci) [reply conanfile.py:22]: still, 1.1.1c:stable is the latest available (https://bintray.com/conan-community/conan/OpenSSL%3Aconan)

**Pull request closed without merging**